diff --git a/usr/share/applications/enpass.desktop b/usr/share/applications/enpass.desktop
index 2ca0a92..f2fb87f 100644
--- a/usr/share/applications/enpass.desktop
+++ b/usr/share/applications/enpass.desktop
@@ -7,7 +7,7 @@ GenericName=Enpass Password Manager
 Icon=enpass
 Icon[en_IN]=enpass
 Terminal=false
-Exec=/opt/enpass/Enpass %U
+Exec=/usr/bin/enpass %U
 MimeType=x-scheme-handler/enpassauth;x-scheme-handler/enpasscard;x-scheme-handler/enpassstart;x-scheme-handler/enpass;x-scheme-handler/cloudkit-7adb8cc6tf.in.sinew.walletx;application/enpasscard;
 Categories=Utility
 
